ui/cryptomaterial/editor.go

```diff
@@ -433,9 +436,7 @@ func (e *Editor) handleEvents(gtx C) {
 	}
 
 	if e.paste.Clicked() {
-		if e.eventKey > 0 {
```

Comment:
@dreacot this condition not relate to crash or not
may be new gioui version will fix it, I saw the code fixed it